### DISCUSSION TITLE
Reveal FieldIDs

### DISCUSSION THREAD

**Discussion by jonathanTack** [cortexrd/Knack-Toolkit-Library#335] [Q&A]
I'm going to be extra needy for awhile...!
Is there a way to reveal Field IDs on screen with the "Show IDs" button? 

![image](https://github.com/user-attachments/assets/4df6de08-8bc1-4490-8f91-15b85621173b)

**Comment by cortexrd** [comment]
No, the **View IDs** is just for views.

But if you have the Developer role, you can do **Ctrl+Shift** and **Hover** around to see all the possible details you can imagine (or almost!)

When you release the two buttons, the popup will freeze with the desired element information.

See this video for a demo:  https://www.youtube.com/watch?v=s396vEBKT8Q

Norm

**Comment by cortexrd** [comment]
Note that on very busy pages, this hover-around feature doesn't work too well, becoming very sluggish.  The solution is to **hover first**, THEN to press Ctrl+Click to see the popup.

**Reply by jonathanTack** [reply]
that hover + Ctrl+Shift is actually not working for me. is that a Configurator issue on my end ?

I did, though, discover the obvious - that if i hover on the field header, my browser will display the field number in a context in the lower-left corner: 

![image](https://github.com/user-attachments/assets/43dec9c3-834b-400e-abc4-9af446b605e1)

**Comment by chprojectsupport** [comment]
Hi JT, yes, is a main switch. Carl HolmesOn 2 Aug 2024, at 01:15, jonathanTack ***@***.***> wrote:﻿
that hover + Ctrl+Shift is actually not working for me. is that a Configurator issue on my end ?
I did, though, discover the obvious - that if i hover on the field header, my browser will display the field number in a context in the lower-left corner:
image.png (view on web)
